skywalking-sniffer/skywalking-api/src/main/java/com/a/eye/skywalking/api/Tracing.java

```diff
@@ -21,4 +24,34 @@ public static String getTraceId() {
 
         return formatTraceId(spanData.getTraceId());
     }
+
+
+    /**
+     * Get the latest span of current trace.
+     *
+     * @return span. if it exists, or null
+     */
+    public static Span getLatestSpan() {
```

Comment:
rename to `getActiveSpan` or `getCurrentSpan`. There is no guarantee about, the top span of stack, is the latest span.